n2y/plugins/expandbluetoggles.py

```diff
@@ -9,16 +9,16 @@
 logger = logging.getLogger(__name__)
 
 
-class HiddenJinjaToggleBlock(ToggleBlock):
+class ExpandBlueToggleBlock(ToggleBlock):
     def __init__(self, client, notion_data, page, get_children=True):
         super().__init__(client, notion_data, page, get_children)
-        if self.notion_type_data["color"] != "blue":
+        if self.notion_type_data["color"] != "blue_background":
```

Comment:
We'll need to update the test for this as well: [tests/test_hiddenjinjatoggles.py](https://github.com/innolitics/n2y/pull/102/files#diff-6fd060877e313b38e5a31293414cd3517041c4b9bdb406eefe42c3e20501ddb5)